src/components/Column/Column.jsx

```diff
@@ -30,50 +40,64 @@ class Column extends Component {
   };
 
   addJob() {
-    return "Add Job"
+    return (
+      !this.state.isAddJobClicked ? 
+        <div className="column-addjob" onClick={this.toggleAddJob}>
+          Add Job
+        </div>
+        : 
+        <div>
+          <div className="column-addjob-form">
+            <form id="addjob" >
+              <p contenteditable="true" className="addjob-company" id="company" >Enter Company Name</p>
+              <h1 contenteditable="true" className="addjob-position" id="jobTitle" >Enter Job Title</h1>
+            </form>
+          </div>
+          <div>
+            <button className="column-addjob-button-cancel" onClick={this.toggleAddJob}>Cancel</button>
+            <button className="column-addjob-button-addjob" onClick={this.toggleAddJob}>Add Job</button>
+          </div> 
+        </div>
+          
+    )
   }
 
   generateColumnHeader() {
     return (
-    <div className="column-header-container">
-        <div className="column-header">
-          <div className="column-header column-icon">
-            <img src={this.props.icon}/>
-          </div>
-          <div className="column-header column-title">
-            {this.props.title}
-            ({this.props.totalCount})
-          </div>
-          <div className="column-header column-details">
-            {this.props.details}
-          </div>          
-        </div>
-        <div className="column-addjob">
-          {this.addJob()}
+    <div className="column-header-container"  style= {this.state.isAddJobClicked ? {height:180}:{}} >
```

Comment:
Same as previous comment

Reply:
Might you show me how to create the dynamic css function in .jsx again?